### DISCUSSION TITLE
Friends Group

### DISCUSSION THREAD

**Discussion by rakib201000** [community/community#138396] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

_**It is a friend group🤣**_

**Comment by rakib201000** [comment]
Hi!@Habib2444

**Reply by Habib2444** [reply, marked as answer]
Hy @rakib201000 What's up

**Comment by ebndev** [comment]
Hi @rakib201000, thanks for being a part of the GitHub Community!

In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. This type of post is considered "off topic" as GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. Any repeat and future violations may result in a temporary or indefinite block from the Community. 

We appreciate your understanding.
